src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] colors = Color.values();
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(colors.length);
```

Comment:
Please extract Random as class variable in order not to create new object each time method is called